frontend/src/metabase/dashboard/components/DashboardMoveModal.styled.tsx

```diff
@@ -1,6 +1,12 @@
 import styled from "@emotion/styled";
 
+import Link from "metabase/core/components/Link";
+
 export const ToastRoot = styled.div`
   display: flex;
   align-items: center;
 `;
+
+export const CollectionLink = styled(Link)`
+  color: var(--mb-color-brand);
```

Comment:
I added this color here to make it consistent with Collection link in `QuestionMoveToast`

Reply:
it would be nice if you could use css modules instead